### ISSUE TITLE
Iframe element missing accessible name.

### ISSUE THREAD

**Issue by Sukha11y** [Sukha11y/Test-ZT-fix#4]
Problem
Where is it located
index.html - [https:///c%3A/Users/SukhadaPawar/Test-ZT-fix/Test-ZT-fix/index.html](https://c/Users/SukhadaPawar/Test-ZT-fix/Test-ZT-fix/index.html)

Locate element
XPath or CSS selector
body>div:nth-of-type(6)>*:nth-child(2)>*:nth-child(3)>*:nth-child(1)

Copy selector to clipboard
HTML element
<iframe src="https://example.com" width="100%" height="120"></iframe>

Copy HTML element to clipboard
Description
One or more iframe elements, excluding those elements with aria-hidden='true' or tabindex set to a negative value, not intentionally hidden in the DOM, does not have a mechanism that allows an accessible name value to be calculated.
Why it matters
When frame elements are not identified, it becomes difficult for users of assistive technologies to move between frames or identify the purpose of content in the iframe. Additionally, some screen readers may use other properties to provide a name which may be inaccurate.

Solution
What to do
Provide an accessible name that is a short description of the contents of an iframe (e.g., "Social Media", etc.). Avoid using location based words, such as "top", "bottom", "left", "right" and redundant descriptive words, such as "frame", within the accessible name. Instead, provide meaningful descriptions of the purpose of the iframe.
If the iframe is not user facing, set the tabindex to -1 and still give the frame a title attribute, such as "empty". It may also be useful to set the width and height to 0 to make sure the frame is not detected by screen readers and also set aria-hidden to true.

An accessible name can be set via the title attribute, aria-label, or aria-labelledby attribute referencing text that identifies the iFrame. Exceptions The automatic test related to this test will not flag frames without a title if the frame/iframe has an inline style attribute that contains the text "display:none;". If frames are used to meet WCAG 2.x SC 2.4.1 then failing this test would impact conforming to that criteria.
